src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link, NavLink, withRouter } from 'react-router-dom'
+
+const Navbar = (props) => {
+  return (
+    <nav>
+      <div className="Navbar">
+        <Link className="brand-logo" to="/">Revision.io</Link>
```

Comment:
I don't think we need this className anymore

Reply:
Oh! I see. I changed it.